community.html

```diff
@@ -0,0 +1,32 @@
+---
+layout: default
+title: Community
+---
+
+{% include nav.html active='Community' %}
+
+<section class="front-section">
+  <div class="container">
+    <div class="col-wide">
+
+      <h1>Community</h1>
+
+      <p>
+        The following resources are available for you if you have a question regarding Neovim:
+      </p>
+
+      <ul>
+        <li>Mailing list: <a href="https://groups.google.com/forum/#!forum/neovim">Neovim Google group</a></li>
```

Comment:
Gitter seems to be significantly more active than anything else here, so perhaps it should be at the top.